packages/atomic/cypress/integration/result-list/result-components-v1/result-badge.cypress.ts

```diff
@@ -0,0 +1,120 @@
+import {generateComponentHTML} from '../../../fixtures/test-fixture';
+import {
+  interceptSearchResponse,
+  setUpPage,
+} from '../../../utils/setupComponent';
+import {
+  addTranslations,
+  executeFirstSearch,
+  setFieldCaptions,
+} from '../../search-interface-utils.cypress';
+import {
+  generateResultList,
+  generateResultTemplate,
+  getFirstResult,
+} from '../result-list-v1-selectors';
+import {ResultBadgeSelectors} from './result-badge-selectors';
+
+describe('Result Badge Component', () => {
+  describe('outside of a result template', () => {
+    function setupResultBadgePage(
+      props: Record<string, string | number>,
+      executeSearch = true
+    ) {
+      setUpPage(
+        generateComponentHTML(ResultBadgeSelectors.component, props).outerHTML,
+        executeSearch
+      );
+    }
+
+    function getResultBadge() {
+      return cy.get(ResultBadgeSelectors.component).shadow();
+    }
+
+    describe('with a field', () => {
+      beforeEach(() => {
+        setupResultBadgePage({field: 'title'});
+      });
+
+      it.skip('should remove the component from the DOM', () => {
```

Comment:
Why is there a lot of skipped tests in this file ?

Reply:
Because they aren't passing yet, and I'll fix their related issues in different PRs